elife/nginx-php7.sls

```diff
@@ -34,4 +42,5 @@ php-fpm:
         - name: php7.0-fpm
         - require:
             - file: php-fpm
-            - file: php-fpm-config            
+            - file: php-fpm-config
+            - php-fpm-pool
```

Comment:
probably needs 
```
- watch:
    - php-fpm-pool
```
to be restarted on changes. Probably should also watch the other ones, same reasoning.
Alternative could be to try `reload: True`.

Reply:
Example: https://github.com/Enrise/phpfpm-formula/blob/1b989c5913b3f437100982c73227aaeeb6ed18d0/phpfpm/service.sls#L27-L33

Reply:
Also `enable: True` would make sure it's started on boot.

Reply:
Added.